10-setup-htcondor.sh

```diff
@@ -367,6 +367,20 @@ if is_true "$CONTAINER_PILOT_USE_JOB_HOOK" && [[ ! -e ${prepare_hook} ]]; then
     exit 1
 fi
 
+cat <<EOF
```

Comment:
Can you add some validation? Bail out if $ACCEPT_JOBS_FOR_HOURS or $RETIREMENT_HOURS are not positive integers?

Reply:
Added